demo/src/main/java/com/orange/ods/demo/ui/components/cards/ComponentCard.kt

```diff
@@ -42,12 +45,25 @@ fun ComponentCard(variant: Variant) {
                         text = stringResource(id = R.string.component_element_thumbnail),
                         trailing = OdsSwitchTrailing(checked = thumbnailChecked)
                     )
+                } else if (variant == Variant.CardHorizontal) {
+                    OdsChoiceChipsFlowRow(
+                        selectedChip = imagePosition,
+                        modifier = Modifier.padding(horizontal = dimensionResource(id = R.dimen.screen_horizontal_margin)),
+                        outlinedChips = true
+                    ) {
+                        OdsChoiceChip(textRes = R.string.component_card_horizontal_position_start, value = OdsHorizontalCardImagePosition.Start)
+                        OdsChoiceChip(textRes = R.string.component_card_horizontal_position_end, value = OdsHorizontalCardImagePosition.End)
+                    }
+                    OdsListItem(
```

Comment:
Should we disable the possibility of displaying the divider when there is no button displayed ?